src/Featurizers/Components/RollingWindowTransformerBase.h

```diff
@@ -0,0 +1,163 @@
+// ----------------------------------------------------------------------

```

Comment:
This should live in the Base subdir (which is a sibling of Components)

Reply:
Good point. Relocated the file.